### ISSUE TITLE
Compilation chain of scala-js-actors

### ISSUE THREAD

**Issue by oliviercooper** [sjrd/scala-js-actors#1]
Hello,
I must miss something for compiling scala-js-actors since its recent downgrade to scala 2.10.
When I try to compile scala-js-actors I have 2 dependencies errors : 

[error](akkaWebsocketBridge/*:update) sbt.ResolveException: unresolved dependency: org.scalajs#scalajs-pickling-play-json_2.10;0.1-SNAPSHOT: not found

[error](actors/*:update) sbt.ResolveException: unresolved dependency: org.scalajs#scalajs-pickling_2.10;0.1-SNAPSHOT: not found

I wanted to compile scala-js-pickling locally, but I have another dependency problem : 

[error](playjson/*:update) sbt.ResolveException: unresolved dependency: org.scalajs#scalajs-pickling-core-jvm_2.10;0.1-SNAPSHOT: not found

Do you have an advice (I'd like to play with last commit ;-) ?

Thanks

**Comment by simonlischka** [comment]
Hey guys, I followed the instructions, cloned scala-js-pickling from https://github.com/scala-js/scala-js-pickling, checked out commit 393d7770502a for version 0.3 and ran the commands you specified above.

packageJS runs smoothly, but when executing run, I get 

org.scalajs#scalajs-pickling-play-json_2.10;0.3-SNAPSHOT: not found.

I didn't make any modifications in the build.sbt files.

Do I have to check out a different repo or run a different series of commands for this to work? 

I'm currently setting up a project and wanted to experiment with scalajs-actors for client/server communication as I found the idea very interesting. 

Another question I have if using scala-js-pickling 0.4.0 is somewhat realistic. My first attempts in modifying version numbers again produce a "not found" error as above.

I used:
libraryDependencies += "be.doeraene" %%% "scalajs-pickling" % "0.4.0"
libraryDependencies += "be.doeraene" %% "scalajs-pickling-play-json" % "0.4.0"

In both cases I used sbt ++2.10.3 publishLocal - am I missing something?
